cedar-policy-core/src/ast/policy_set.rs

```diff
@@ -161,7 +161,11 @@ impl PolicySet {
     }
 
     /// Add a template to the policy set.
+    /// If a link with the same name already exists, this will error.
```

Comment:
what about if a static policy with the same name already exists?  or if a template with the same name already exists?  I'm sure these are both errors, but we can/should call that out in the doc comment

Reply:
Fair point.